src/defs/jsonNode.ts

```diff
@@ -0,0 +1,40 @@
+/**
+ * https://www.keycloak.org/docs-api/15.0/rest-api/#_jsonnode
+ */
+
+enum JsonNodeType {
+  ARRAY,
+  BINARY,
+  BOOLEAN,
+  MISSING,
+  NULL,
+  NUMBER,
+  OBJECT,
+  POJO,
+  STRING,
+}
+
+export default interface JsonNode {
```

Comment:
I believe that this is not actually the correct type that the API returns, although it is documented this way. It looks like the returned type is actually a JSON object that has no predefined structure. Perhaps the type `object` would be more appropriate for these fields as that matches it better.

I think the type from the documentation is some kind of internal type that is leaking through.

Reply:
Good call - thank you. Using `object` here also makes sense